### PR TITLE
change the href attribute of learn more button

### PR DESCRIPTION
**Pull request by pritam825**
Pull Request for Issue #36204  .

### Summary of Changes
The "learn more" button on the scheduled tasks empty state currently points to the gsoc github repositary. It should point to the docs site link

### Testing Instructions
go to scheduled tasks empty state , click on learn more

### Actual result BEFORE applying this Pull Request

After clicking on the learn more button it is going to the gsoc github repositary.

### Expected result AFTER applying this Pull Request

After clicking on the learn more button It should point to the docs site link

### CODE REVIEW

**Comment by brianteeman** [comment]
This just points to the hompege and not a page with specific information

**Comment by pritam825** [comment]
@brianteeman  now can you please check again, Thanks for giving me your valuable time  :)

**Comment by brianteeman** [comment]
please look at the other similar links

**Comment by pritam825** [comment]
@brianteeman I am trying to find the similar links, but not able to find, still trying can you please suggest and help from where I can get this, thanks :)

File: administrator/components/com_scheduler/tmpl/tasks/empty_state.php [hunk 1]
```diff
@@ -15,7 +15,7 @@
 $displayData = [
 	'textPrefix' => 'COM_SCHEDULER',
 	'formURL' => 'index.php?option=com_scheduler&task=task.add',
-	'helpURL' => 'https://github.com/joomla-projects/soc21_website-cronjob',
+	'helpURL' => 'https://docs.joomla.org/J4.x:Task_Scheduler',
```
Comment (brianteeman) [inline administrator/components/com_scheduler/tmpl/tasks/empty_state.php]:
```suggestion
	'helpURL' => 'https://docs.joomla.org/Special:MyLanguage/J4.x:Task_Scheduler',
```
This addition means that the docs site will automatically change to the users language

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 06b3ab8a180c9ef82286b31d547d104f2f1d3e59<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36666">issues.joomla.org/tracker/joomla-cms/36666</a>.</sub>

**Comment by pritam825** [comment]
@brianteeman Thanks :)

**Comment by Shubhamverma2796** [comment]
I have tested this item :white_check_mark: successfully on 06b3ab8a180c9ef82286b31d547d104f2f1d3e59<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36666">issues.joomla.org/tracker/joomla-cms/36666</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36666">issues.joomla.org/tracker/joomla-cms/36666</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by pritam825** [comment]
Thanks for merging

**Pull request merged**